cookbook/doctrine/registration_form.rst

```diff
@@ -211,11 +211,9 @@ Next, create the form for this ``Registration`` model::
         public function buildForm(FormBuilderInterface $builder, array $options)
         {
             $builder->add('user', new UserType());
-            $builder->add(
-                'terms',
-                'checkbox',
-                array('property_path' => 'termsAccepted')
-            );
+            $builder->add('terms', 'checkbox', array(
+                'property_path' => 'termsAccepted',
```

Comment:
I would rather name the field ``termsAccepted`` and then use the ``label`` option to set a better label rather than using ``property_path`` to change the property path to a different one than the default. It is simpler to understand IMO